weave-js/src/components/PagePanelComponents/Home/Browse3/pages/OpVersionPage.tsx

```diff
@@ -16,256 +15,120 @@ import {
-                project={project}
-                callCount={opVersionCallCount}
-                filter={{
-                  opVersionRefs: [opVersion.refUri()],
-                }}
-                neverPeek
-                variant="secondary"
-              />
-            ),
-            ...(opVersionCategory
+              ) : (
+                <></>
+              ),
+            ...(category
               ? {
-                  Category: <CategoryChip value={opVersionCategory} />,
+                  Category: <CategoryChip value={category} />,
                 }
               : {}),
-
-            // Dropping input types and output types for the time being since
-            // we have de-prioritized type version navigation.
-            // ...(opInputTypes.length > 0
-            //   ? {
-            //       'Input Types': (
-            //         <ul style={{margin: 0}}>
-            //           {opInputTypes.map((t, i) => (
-            //             <li key={i}>
-            //               <TypeVersionLink
-            //                 entityName={t.entity()}
-            //                 projectName={t.project()}
-            //                 typeName={t.type().name()}
-            //                 version={t.version()}
-            //               />
-            //             </li>
-            //           ))}
-            //         </ul>
-            //       ),
-            //     }
-            //   : {}),
-            // ...(opOutputTypes.length > 0
-            //   ? {
-            //       'Output Types': (
-            //         <ul style={{margin: 0}}>
-            //           {opOutputTypes.map((t, i) => (
-            //             <li key={i}>
-            //               <TypeVersionLink
-            //                 entityName={t.entity()}
-            //                 projectName={t.project()}
-            //                 typeName={t.type().name()}
-            //                 version={t.version()}
-            //               />
-            //             </li>
-            //           ))}
-            //         </ul>
-            //       ),
-            //     }
-            //   : {}),
-            ...(opInvokes.length > 0
-              ? {'Call Tree': <OpVersionOpTree opVersion={opVersion} />}
-              : {}),
           }}
         />
       }
       tabs={[
         {
           label: 'Code',
           content: (
-            // <Box
-            //   sx={{
-            //     height: '100%',
-            //     width: '100%',
-            //     flexGrow: 1,
-            //     overflow: 'hidden',
-            //     pt: 4,
-            //   }}>
             <OpCodeViewer
               entity={entity}
               project={project}
-              opName={opName}
-              opVersions={opVersions.slice().reverse()} // put in increasing order
+              opName={opId}
+              opVersions={opVersions.result ?? []} // put in increasing order
```

Comment:
See the comment - the diff viewer is expecting versions in increasing order, so if you remove the reverse here you'll break it.

Reply:
On this one, the data returned from the server is already in the correct order for display. I manually verified as well.

Reply:
ok, scratch that. I thought you wanted the most recent/current on left side. Switching back

Reply:
👍

Reply:
You are changing the order from what we have in prod.

This PR:
<img width="281" alt="Screenshot 2024-02-08 at 10 13 36 AM" src="https://github.com/wandb/weave/assets/112953339/7f261971-d7bb-4440-b18f-674e798521e7">
<img width="1104" alt="Screenshot 2024-02-08 at 10 13 31 AM" src="https://github.com/wandb/weave/assets/112953339/73019c01-e949-46c2-afce-b2ac924fd15e">

Prod:
<img width="273" alt="Screenshot 2024-02-08 at 10 13 21 AM" src="https://github.com/wandb/weave/assets/112953339/d30e732d-ee44-42a6-9b2d-ea24827172ed">
<img width="1107" alt="Screenshot 2024-02-08 at 10 13 15 AM" src="https://github.com/wandb/weave/assets/112953339/8da35fcc-867e-46fd-abc7-a4c715f6b830">

We could reasonably decide that the selector should  be date decreasing, but I definitely feel like the proper order for the diff is older version on the left, and the previous/next buttons wouldn't do what they say after this either.

Reply:
👍 thanks for the details - I switched it back - my mistake